tee-worker/litentry/core/assertion-build/src/dynamic/contracts/A6.sol

```diff
@@ -18,7 +18,7 @@
 
 pragma solidity ^0.8.8;
 
-import "https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.9.0/contracts/utils/Strings.sol";
+import "./openzeppelin/Strings.sol";
```

Comment:
Any benefit to hard code this import instead keeping them imported during build time?

Reply:
Contracts import via http cannot be decoded and compiled by any compiler, so the contract will be flattened here.
This is supported in remix, but not in solc-related compilers.

Reply:
> so the contract will be flattened here
How will flattened break the solc-related complier? Will it break hardhat compiler too?
I feel if we keep import from source it will be much easier to maintain. So trying to see if any workaround to not to  copy them in our own repo.